tutorials/beginner/uis.md

```diff
@@ -6,23 +6,24 @@ layout: tutorial-beginner
 
 # OpenHAB 2 UIs
 
-OpenHAB 2 offers different UIs in its standard configuration: the _Paper UI_, the _Basic UI_ and the _Classic UI_ (this has to be installed manually from within Paper UI or via config file).
+OpenHAB 2 offers different UIs in its standard configuration: the *Paper UI*, the *Basic UI* and the *Classic UI* (this has to be installed manually from within Paper UI or via config file).
 
 ## The Paper UI
 
 The Paper UI is a new interface that helps setting up and configuring your openHAB instance.
 It does not (yet) cover all aspects, so you still need to resort to textual configuration files, but it already offers the following:
 
-- Add-on management: Easily install or uninstall openHAB add-ons
+-   Add-on management: Easily install or uninstall openHAB add-ons
```

Comment:
Are those extra spaces needed? This comments addresses the following list items until line 23, too.

Reply:
Hi @Confectrician, according to the linter, you need two spaces after the dash mark.  I am assuming that @ThomDietrich has established this as the style he wants, so I just adjusted the text to get rid of the error.  Is that okay?

Reply:
I still had luck yet with setting up my linter environment. 😄 
So if the rules say that, its ok for me and i would approve and merge the change then, if you are ready from your side.

Reply:
Yes please go ahead - the linter rules definitely say that the extra spaces should be there.  I checked this again last night when I saw your comment.